packages/wp-story-editor/src/api/story.js

```diff
@@ -28,11 +29,7 @@ import apiFetch from '@wordpress/api-fetch';
  * Internal dependencies
  */
 import { STORY_EMBED, STORY_FIELDS } from './constants';
-import {
-  base64Encode,
-  transformStoryResponse,
-  snakeToCamelCaseObjectKeys,
-} from './utils';
+import { base64Encode, transformStoryResponse } from './utils';
```

Comment:
Should we move `base64Encode`, `transformStoryResponse` to `@web-stories-wp/wp-utils`?

Reply:
They are only used by `wp-story-editor` and not shared with `wp-dashboard`. So no need to move to a shared package. That'd just make the `wp-dashboard` bundle bigger for no benefit.

Reply:
Would it make the bundle bigger?

Reply:
Well the tree shaking might be stripping things out again, but I'm not betting on that with webpack v4 :)

Still, moving stuff that isn't actually shared to a shared package does not achieve anything